components/Common/Button/index.module.scss

```diff
@@ -0,0 +1,59 @@
+.button {
+  @apply px-[18px] py-[10px] font-semibold;
+
+  &.primary {
+    @apply bg-green-600 text-white rounded border border-green-600 shadow-sm;
+
+    &:hover {
+      @apply bg-green-700 border-green-700;
+    }
+
+    &:focus {
+      @apply bg-green-700 border-green-700;
+    }
+
+    &:disabled {
+      @apply bg-green-600 opacity-50;
+    }
+
+    &.special {
+      @apply bg-neutral-100/0 bg-gradient-to-r from-green-600/30 via-green-500/20 to-green-600/10 shadow-sm border border-green-600/30;
```

Comment:
Can you separate the tailwind selectors to one per line? As shown on the Collaborator Guide? Eg.:


```scss
@apply bg-neutral-100/0
 bg-gradient-to-r
 from-green-600/30
 ...
 border-green-600/30;
```

Reply:
Do that on all the `@apply` calls

Reply:
Sure, I'll do that